Libs/Gtk-3.0/Classes/Image.cs

```diff
@@ -0,0 +1,13 @@
+﻿using GObject.Native;
+
+namespace Gtk
+{
+    public partial class Image
+    {
+        public static Image New()
+            => ObjectWrapper.WrapHandle<Image>(Native.Image.Instance.Methods.New(), false);
```

Comment:
The wrapper checks if an object is already known for this pointer and afterwards calls the constructor per reflection. But as we are creating a new instance we can omit this check as we now there is no other instance and direcly call a constructor.